pkg/aggregate/aggregate.go

```diff
@@ -122,16 +122,22 @@ func AggregatorsToStringList(aggregators string) ([]string, error) {
 // convert comma separated string to aggregator mask
 func AggrsFromString(split []string) (AggrType, error) {
 	var aggrList AggrType
+	var hasAggregators bool
 	for _, s := range split {
 		trimmed := strings.TrimSpace(s)
 		if trimmed != "" {
 			aggr, ok := aggrTypeString[trimmed]
 			if !ok {
 				return aggrList, fmt.Errorf("invalid aggragator type '%s'", s)
 			}
+			hasAggregators = true
 			aggrList = aggrList | aggr
 		}
 	}
+	// Always have count aggregator by default
```

Comment:
@talIguaz  @igo isnt it simpler to just check if aggrList != 0 ?

Reply:
the slice of strings here is created after calling `strings.split(<agg>, ',')` so if someone by mistake or whatever entered aggregators as let's say `  , ` , the slice will not be empty but there also will not be any aggregators